content/en/docs/community/release_checklist.md

```diff
@@ -165,12 +200,14 @@ need to update corresponding tests that are using that version number.
 
 ```shell
 git add .
-git commit -m "bump version to $RELEASE_CANDIDATE_NAME"
+git commit -m "bump version to $RELEASE_NAME"
 ```
 
 This will update it for the $RELEASE_BRANCH_NAME only. You will also need to
 pull this change into the master branch for when the next release is being
-created.
+created, as in [this example of 3.2 to
+3.3](https://github.com/helm/helm/pull/8411/files), and add it to the milestone
+for the next release.
```

Comment:
I thought that showing an instance of this update would provide a useful reference.